### DISCUSSION TITLE
[HELP] SEB shows me the error 'please remove any external webcam'

### DISCUSSION THREAD

**Discussion by harshunchkin** [SafeExamBrowser/seb-win-refactoring#961] [Q&A]
Hi, I am a student who recently gave an exam on SEB. The issue is that my integrated webcam is no longer working so I bought an external one (Logitech C270 Webcam). But during the exam I was logged out multiple times as the SEB detected two cameras, and it showed me the list 1. Integrated webcam 2. Logitech c270 hd webcam. Is there a way I can remove the integrated web camera so that in the future exams I don't get these errors

**Comment by dbuechel** [comment]
SEB does not restrict the amount or type of connected cameras. This likely is a restriction in the LMS / web application you were using for the exam.

**Reply by harshunchkin** [reply]
Does that mean that I need to use another web-browser? Or do I need to tell the exam admin to remove the restriction?

**Reply by dbuechel** [reply]
Yes, you'll need to contact the exam administrator and resolve the issue with them.
